lib/core/src/lib.rs

```diff
@@ -13,3 +13,160 @@ pub(crate) mod send_swap;
 pub(crate) mod swapper;
 pub(crate) mod utils;
 pub(crate) mod wallet;
+
+pub use sdk_common::prelude::*;
+
+// === FRB mirroring
+//
+// This section contains frb "mirroring" structs and enums.
+// These are needed by the flutter bridge in order to use structs defined in an external crate.
+// See <https://cjycode.com/flutter_rust_bridge/v1/feature/lang_external.html#types-in-other-crates>
+
+use flutter_rust_bridge::frb;
```

Comment:
Can we move these to the binding.rs same as we did with the sdk?

Reply:
Done: db821d9b4ff20c00d4aef8ea51558d011c5490fd